### PR TITLE
Correct plural, singular forms #244

### PR DESCRIPTION
**Pull request by asadafasab**
Fixing plural and singular forms.
I used multi-line string because of the line length.

### CODE REVIEW

**Comment by asadafasab** [comment]
example

**Comment by darrenburns** [comment]
Thanks for this!

I think we could probably avoid using multiline strings here if we extract some variables similar to this:

```python
        test_plural = "test" if self.num_tests_collected == 1 else "tests"
        fixture_plureal = "fixture" if self.num_fixtures_collected == 1 else "fixtures"
```

and then inject those into the f-string.

We can also change the `[/b]` to `[/]`. It's a shorthand offered by the Rich library for closing tags.

Let me know if you have any questions :)

**Comment by asadafasab** [comment]
Thanks, but for coherence, I'll stick to `[/b]`.

File: ward/_terminal.py [hunk 1]
```diff
@@ -277,9 +277,12 @@ def __rich_console__(self, c: Console, co: ConsoleOptions) -> RenderResult:
                 path = self.config_path.name
             yield f"Loaded config from [b]{path}[/b]."
 
+        test_plural = "test" if self.num_tests_collected == 1 else "tests"
+        fixture_plureal = "fixture" if self.num_fixtures_collected == 1 else "fixtures"
```
Comment (darrenburns) [inline ward/_terminal.py:281]:
```suggestion
        fixture_plural = "fixture" if self.num_fixtures_collected == 1 else "fixtures"
```

File: ward/_terminal.py [hunk 2]
```diff
@@ -277,9 +277,12 @@ def __rich_console__(self, c: Console, co: ConsoleOptions) -> RenderResult:
...
+
         yield (
-            f"Found [b]{self.num_tests_collected}[/b] tests "
-            f"and [b]{self.num_fixtures_collected}[/b] fixtures "
+            f"Found [b]{self.num_tests_collected}[/b] {test_plural} "
+            f"and [b]{self.num_fixtures_collected}[/b] {fixture_plureal} "
```
Comment (darrenburns) [inline ward/_terminal.py:285]:
```suggestion
            f"and [b]{self.num_fixtures_collected}[/b] {fixture_plural} "
```

**Review by darrenburns — CHANGES_REQUESTED** [review]

**Comment by asadafasab** [comment]
Sorry

**Comment by darrenburns** [comment]
Thanks for the contribution! I'm not sure why the Codecov report is failing. It's been happening all day and isn't related to your changes.

**Pull request merged**